requirements.txt

```diff
@@ -6,3 +6,4 @@ pycollada
 typing-extensions
 pycryptosat!=5.11.23
 python-sat[cryptosat]
+tqecd @ git+https://github.com/tqec/tqecd.git
```

Comment:
I'll create a tag on the `tqecd` repository, we should use that tag. Not using the tag might be an issue because the code will likely change, and we would like to avoid breaking the `tqec` package due to changes in `tqecd`.